### PR TITLE
Add quickstart Puppet management guide

### PR DESCRIPTION
**Pull request by domcleal**
Matches the screencast.

### CODE REVIEW

**Comment by skottler** [comment]
This also looks quite good. :+1:

File: _includes/manuals/1.2/2.2_quickstart_puppet.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Change back to the *Hosts* tab and click *Edit* on the Foreman host.  On the *Puppet Classes* tab, expand the *ntp* module and click the + icon to add the *ntp* class to the host, then save the host.
+
+<div class="alert alert-info">Managed parameters can be overridden when editing an individual host from its <i>Parameters</i> tab.</div>
+
+Clicking the *YAML* button when back on the host page will show the *ntp* class and the server_list parameter, as passed to Puppet via the ENC (external node classifier) interface.  Re-run `puppet agent --test` on the Foreman host to see the NTP service automatically reconfigured by Puppet and the NTP module.
+
+#### Adding more Puppet-managed hosts
+
+Other hosts with Puppet agents installed can use this puppet master by setting `server = foreman.example.com` in puppet.conf.  Sign their certificates in Foreman by going to *More > Configuration > Smart Proxies > Certificates* or using `puppet cert list` and `puppet cert sign` on the puppet master.
```
Comment (lzap) [inline _includes/manuals/1.2/2.2_quickstart_puppet.md:39]: Other hosts = Other hosts which are not managed by Foreman?

Nice!

Comment (domcleal) [inline _includes/manuals/1.2/2.2_quickstart_puppet.md:39]: Yes, should I word this better?

Comment (lzap) [inline _includes/manuals/1.2/2.2_quickstart_puppet.md:39]: Nah, I think this is fine.

LZ

On Tue, Aug 27, 2013 at 01:40:35AM -0700, Dominic Cleal wrote:

> > +#### Using the Puppet module
> > +
> > +Click on the "ntp" class in the list, change to the _Smart Class Parameters_ tab and select the _server_list_ parameter on the left hand side.  Tick the _Override_ checkbox so Foreman manages the "server_list" parameter of the class and change the default value if desired, before submitting the page.
> > +
> > +\* More info: [Parameterized classes documentation](/manuals/1.2/index.html#4.2.5ParameterizedClasses)
> > +\* Screencast: [Parameterized class support in Foreman](http://www.youtube.com/watch?v=Ksr0tilbmcc)
> > +
> > +Change back to the _Hosts_ tab and click _Edit_ on the Foreman host.  On the _Puppet Classes_ tab, expand the _ntp_ module and click the + icon to add the _ntp_ class to the host, then save the host.
> > +
> > +<div class="alert alert-info">Managed parameters can be overridden when editing an individual host from its <i>Parameters</i> tab.</div>
> > +
> > +Clicking the _YAML_ button when back on the host page will show the _ntp_ class and the server_list parameter, as passed to Puppet via the ENC (external node classifier) interface.  Re-run `puppet agent --test` on the Foreman host to see the NTP service automatically reconfigured by Puppet and the NTP module.
> > +
> > +#### Adding more Puppet-managed hosts
> > +
> > +Other hosts with Puppet agents installed can use this puppet master by setting `server = foreman.example.com` in puppet.conf.  Sign their certificates in Foreman by going to _More > Configuration > Smart Proxies > Certificates_ or using `puppet cert list` and `puppet cert sign` on the puppet master.
> 
> Yes, should I word this better?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/theforeman/theforeman.org/pull/78/files#r5999909

## 

Later,

 Lukas "lzap" Zapletal
 irc: lzap #theforeman

**Comment by domcleal** [comment]
Merged, thanks for the review!

**Pull request closed without merging**